frontend/src/components/v2/Input/Input.tsx

```diff
@@ -116,3 +120,30 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
 );
 
 Input.displayName = "Input";
+
+export type PasswordInputProps = Omit<InputProps, "type" | "rightIcon">;
+
+export const PasswordInput = forwardRef<HTMLInputElement, PasswordInputProps>(
+  (props, ref): JSX.Element => {
+    const [showPassword, setShowPassword] = useToggle(false);
+
+    return (
+      <Input
+        {...props}
+        ref={ref}
+        type={showPassword ? "text" : "password"}
+        rightIcon={
+          <button
```

Comment:
Can you please change this to IconButton component

Reply:
Done 👍🏼